CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
+
+### Fixes
+
+- Network request sentry-trace propagation incorrectly assign (#2167)
```

Comment:
This is kind of an indecipherable sentence 😅

Reply:
Suggestions are welcome 😁